File: submissions/sm_016_manish-sharma/week_3/day_4/session_1/mirrorImage.js
function mirrorImage(input){
   var output=''
   arr1=['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
   arr2=['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']


   for(i=0;i<input.length;i++){                      
        for(j=0;j<arr1.length;j++){                   //checking for input element in array 1
           if (input[i]===arr1[j]){
               c=(Number((arr1.length-1)-j))
               output += arr1[c]                      //pushing the mirror image element into output string
           }
           else if (input[i]===arr2[j]){              //checking for input element in array 2
            c=(Number((arr2.length-1)-j))
            output += arr2[c]                        ////pushing the mirror image element into output string
           }
        }
    }
    console.log(output)
}

mirrorImage('AyzB')

File: submissions/sm_105_ashish/week_13/day_4/session_2/mirror_image.py
input = "AyzB"
def convert(n):
    if n <=90 and n >= 65:
        return 90 + 65 - n
    if n <= 122 and n >= 97:
        return 122 + 97 - n 
mirror_image = ""
for i in input:
    x = chr(convert(ord(i)))
    mirror_image += x
print(mirror_image)  
  
